awx/main/tasks.py

```diff
@@ -1179,11 +1179,17 @@ def final_run_hook(self, instance, status, private_data_dir, fact_modification_t
         instance.log_lifecycle("finalize_run")
         job_profiling_dir = os.path.join(private_data_dir, 'artifacts/playbook_profiling')
         awx_profiling_dir = '/var/log/tower/playbook_profiling/'
+        collections_info = os.path.join(private_data_dir, 'artifacts/')
```

Comment:
```suggestion
        collections_info = os.path.join(private_data_dir, 'artifacts', 'collections.json')
```

Reply:
The `/artifacts` directory will _always_ exist, but `collections.json` might not. So we should look for that.

Reply:
Yep, makes sense!